pkg/server/commands/read.go

```diff
@@ -76,7 +84,12 @@ func (q *ReadQuery) Execute(ctx context.Context, req *openfgav1.ReadRequest) (*o
 	opts := storage.ReadPageOptions{
 		Pagination: storage.NewPaginationOptions(req.GetPageSize().GetValue(), string(decodedContToken)),
 	}
-	tuples, contToken, err := q.datastore.ReadPage(ctx, store, tupleUtils.ConvertReadRequestTupleKeyToTupleKey(tk), opts)
+	tuples, contUlid, err := q.datastore.ReadPage(ctx, store, tupleUtils.ConvertReadRequestTupleKeyToTupleKey(tk), opts)
+	if err != nil {
+		return nil, serverErrors.HandleError("", err)
+	}
+
+	contToken, err := q.tokenSerializer.Serialize(contUlid, "")
```

Comment:
Where is the call to `tokenSerializer.Deserialize`? 🤔

Reply:
Also, the existing `Serialize` method is only applicable to ReadChanges API, it seems wrong to use it here for Read.

Reply:
oh wait, you're right; too much copy+paste I guess. I wonder how tests didn't catch that!

Reply:
Haha what tests? 😆 we don't have any. `pkg/server/commands/read_test.go` doesn't exist